docs/migration_guide.rst

```diff
@@ -1,3 +1,84 @@
+0.4.8 Migration guide
+=====================
+
+0.4.8 of starknet.py brings fresh changes including a breaking change in contracts' code.
```

Comment:
Maybe we should mention that this breaking change doesn't come from starknet.py?